components/inquiry_questionnaire/map/decisions.coffee

```diff
@@ -4,6 +4,10 @@ hasSeen = (steps...) -> ({ logger }) -> logger.hasLogged steps...
 hasSeenThisSession = (steps...) -> ({ logger }) -> logger.hasLoggedThisSession steps...
 
 decisions =
+  is_auction: ({ artwork }) ->
+    artwork.related()
+      .partner.get('type') is "Auction"
```

Comment:
If I understand correctly, the logic to display the "Ask a specialist" link (and hence trigger the inquiry flow discussed here) is  to check [`artwork.is_in_auction`](https://github.com/artsy/force/blob/dacb41a7c51bc2085b97150b811bb17e7e01970c/apps/artwork/components/auction/index.jade#L1), and it's fetched from [metaphysics](https://github.com/artsy/metaphysics/blob/e5595a7dfe831996b2360c9717f2a7ccf66b4a3b/schema/artwork/index.js#L170). I feel it would be ideal to make them consistent, and, in fact, I am not sure about the relationship between that and the partner type.

Don't want to block this PR but would like to mention it.